docs/reference/components/sinks/datadog_metrics.cue

```diff
@@ -55,16 +55,6 @@ components: sinks: datadog_metrics: {
 	configuration: {
 		api_key:  sinks._datadog.configuration.api_key
 		endpoint: sinks._datadog.configuration.endpoint
-		namespace: {
```

Comment:
Should this have been replaced with `default_namespace`?

Reply:
It was optional before, and I haven't found any requirements for namespace in DataDogs metric submission documentation. Although it seams as a common practice to have it. The original issue [mentioned](https://github.com/timberio/vector/pull/967#discussion_r333456047) this as 
> I'd like to encourage users to fill the namespace config this way, because it's a really useful practice.

So yes, it seams be good to have it as optional in this sink to serve as a nudge. I'll also check if this applies to other sinks.